mods/tuxemon/maps/spyder_bedroom.tmx

```diff
@@ -76,7 +76,7 @@
   </object>
   <object id="28" name="Give Starting Money" type="event" x="32" y="16" width="16" height="16">
    <properties>
-    <property name="act30" value="variable_math money,+,1000000"/>
+    <property name="act30" value="transfer_money +,10000,mom"/>
```

Comment:
Mom will have negative money after this... Use set_money instead?

Reply:
It makes sense, I'll fix